jobs/JREGIONAL_NEXUS_GFS_SFC

```diff
@@ -94,7 +97,10 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-DATA="${DATA:-${COMIN}${SLASH_ENSMEM_SUBDIR}/tmp_NEXUS_GFS_SFC}"
+# DATA="${DATA:-${COMIN}${SLASH_ENSMEM_SUBDIR}/tmp_NEXUS_GFS_SFC}"
+DATA=${DATAROOT}/aqm_nexus_gfs_sfc_${cyc}.${share_pid}
+mkdir -p ${DATA}
```

Comment:
@lgannoaa, this line is just for the `community` mode. In the `nco` mode, NEXUS_GFS_SFC's data directory will not removed even though it is completed and the data directory will be removed in the `run_post` task. You don't have to create a separate data directory for it.

Reply:
This job in one of the four jobs that require DATA to not be removed after complete. This job is also the first job to start the AQM workflow. Therefore a mkdir -p seem to be ok here.

Reply:
I don't worry about 'mkdir'. My concern is `aqm_nexus_gfs_sfc_{cyc}.{share_pid}. This data directory is not necessary.